phonenumbers_test.go

```diff
@@ -1230,9 +1230,11 @@ type timeZonesTestCases struct {
 }
 
 type prefixMapTestCases struct {
-	num      string
-	lang     string
-	expected string
+	num               string
+	lang              string
+	expectedCarrier   string
+	expectedPrefix    int
+	expectedGeocoding string
```

Comment:
Realize this isn't your cruft but in the interest of leaving things nicer than we found them why don't we get rid of this shared test case struct and just use inline/anonymous structs when defining the test cases? That way we don't have to have weird naming and empty values when not relevant.

Otherwise looks great, thank you!

Reply:
Thought about it but wanted to keep changes to a minimum. Let me refactor it 👍🏻

Reply:
Thanks, appreciate it!

Reply:
Done in 04f5062.